omero/sysadmins/version-requirements.txt

```diff
@@ -27,8 +27,8 @@ Operating systems
 
 * Microsoft Windows
 
-  * Proposal: [5.1] 7 (client) recommended / Server 2008R2 (server)
-    recommended; XP/Server 2003/Vista dropped
+  * Proposal: [5.1] XP/Server 2003/Vista dropped, 7 (client)
+    recommended / Server 2008R2 (server) recommended
   * Proposal: [5.2] 7 / Server 2008R2 supported; 8 (client)
```

Comment:
Looks to me like this have a comma rather than a semi-colon to be consistent with the rest